account_financial_risk/models/res_partner.py

```diff
@@ -145,6 +147,21 @@ class ResPartner(models.Model):
     risk_currency_id = fields.Many2one(
         comodel_name="res.currency", compute="_compute_credit_currency"
     )
+    date_credit_limit = fields.Date(
+        compute="_compute_date_credit_limit",
+        store=True,
+        readonly=True,
```

Comment:
Maybe this field should be writable. (keeping the rest of the functionality)
What happens if we want to save the date in which we have changed the conditions and that date has already passed?
In Spain, it is quite common for the administration department to introduce the modifications with some delay. :wink: 
```suggestion
        readonly=False,
```